### PR TITLE
UR5 Robot example

### PR DESCRIPTION
**Pull request by lundjack**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Package: geomstats
```
Comment (ninamiolane) [inline DESCRIPTION]: Once we merge PR #2 , you should 'rebase' your branch on r-geomstats master.

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

File: ur5_example.R [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers
```
Comment (ninamiolane) [inline ur5_example.R]: Address @johmathe 's comments on PR #1 : for example spaces after #
and please follow https://google.github.io/styleguide/Rguide.xml

File: ur5_example.R [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers
+ur5=read.csv(file="ur5testresult_halfspeed_payload1.6lb_1 (1).csv",header=FALSE)
+ur5headers=scan("ur5testresult_header.csv", sep=',', what = "", quiet = TRUE)
+colnames(ur5)=ur5headers
+
+
+install.packages("scatterplot3d")
+library("scatterplot3d")
```
Comment (ninamiolane) [inline ur5_example.R]: No `library` in the middle of the script. In general, please look at the scripts' organization conventions:

From: https://google.github.io/styleguide/Rguide.xml
General Layout and Ordering
If everyone uses the same general ordering, we'll be able to read and understand each other's scripts faster and more easily.

Copyright statement comment
Author comment
File description comment, including purpose of program, inputs, and outputs
source() and library() statements
Function definitions
Executed statements, if applicable (e.g., print, plot)
Unit tests should go in a separate file named originalfilename_test.R.

File: ur5_example.R (hunk 2 above)
Comment (ninamiolane) [inline ur5_example.R]: Please rename the file to something understandable to everybody: for example: `robot_arm_example.R`?

File: ur5_example.R [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers
+ur5=read.csv(file="ur5testresult_halfspeed_payload1.6lb_1 (1).csv",header=FALSE)
```
Comment (ninamiolane) [inline ur5_example.R]: From: https://google.github.io/styleguide/Rguide.xml

"Spacing
Place spaces around all binary operators (=, +, -, <-, etc.). "

To automatically enforce R style convention, please download a "linter", e.g. https://cran.r-project.org/web/packages/lintr/README.html

File: robot_arm_example.R [hunk 5]
```diff
@@ -0,0 +1,13 @@
+library("scatterplot3d")
+
+# read csv data, add headers
+ur5 <- read.csv(file = "ur5testresult_halfspeed_payload1.6lb_1 (1).csv", header = FALSE)
```
Comment (johmathe) [inline robot_arm_example.R]: make this a regular filename

File: NAMESPACE [hunk 6]
```diff
@@ -0,0 +1,2 @@
+# Generated by roxygen2: do not edit by hand
```
Comment (johmathe) [inline NAMESPACE]: why is this needed?

File: DESCRIPTION [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Package: geomstats
+Type: Package
+Title: geomstats for R
+Version: 1.1.0
+Author: John Lund
```
Comment (johmathe) [inline DESCRIPTION]: These two should be the same:

John Lund <jrlund@stanford.edu>
